File: backend/auth-service/src/main/java/com/example/authservice/member/service/MemberService.java
package com.example.authservice.member.service;

import com.example.authservice.member.client.MemberFeignClient;
import com.example.authservice.member.dto.MemberDto;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class MemberService {

    private final MemberFeignClient memberFeignClient;

    public MemberDto getMember(String userId) {
        return memberFeignClient.getMember(userId);
    }
}

File: backend/auth-service/src/main/kotlin/com/example/authservice/member/service/MemberService.kt
package com.example.authservice.member.service

import com.example.authservice.member.client.MemberFeignClient
import com.example.authservice.member.dto.MemberDto
import org.springframework.stereotype.Service

@Service
class MemberService(private val memberFeignClient: MemberFeignClient) {

    fun getMember(userId: String): MemberDto? = memberFeignClient.getMember(userId)
}
